src/simulations/collect.jl

```diff
@@ -510,3 +514,20 @@ collect_model_data!(df, model, properties::Nothing, step::Int = 0; kwargs...) =
 ###################################################
 # Parallel / replicates
 ###################################################
+
+
+###################################################
+# ProgressMeter 
+###################################################
+
+"""
+Show number of passed steps when `n` in `run!` is a function.
+"""
+function next!(p::Vector{Int})
```

Comment:
This is type piracy and definitely not something we can do in a Julia package. It extends a method whose both function, and type of input arguments, we do not own. The ProgressMeter.jl package says how we should use it by initializing a `Progress`.